zerver/management/commands/deliver_email.py

```diff
@@ -39,8 +39,7 @@ class Command(BaseCommand):
     def handle(self, *args: Any, **options: Any) -> None:
 
         if settings.EMAIL_DELIVERER_DISABLED:
-            while True:
-                time.sleep(10*9)
+            time.sleep(10**9)  # "infinity seconds"
```

Comment:
I guess I just merged this, but removing the `time.sleep` was probably not the right approach here.

Reply:
Let's just make this be a call to `zerver.lib.management.sleep_forever` that can do the `white True` thing and have `# nocoverage` on it.  I think that's better.

Reply:
Opened as https://github.com/zulip/zulip/pull/10322